Dockerfile.db

```diff
@@ -0,0 +1,3 @@
+FROM mdillon/postgis:10
```

Comment:
In the install guide there is Postgres 9.6. Although locally I am running on Postgres 10.

Reply:
Indeed, there should not be anything preventing CyclOSM to be run with PostgreSQL > 9.6. The INSTALL.md guide might night an update (maybe use the generic `postgresql` package without mentionning versions).